app/containers/HomeContainer/index.js

```diff
@@ -6,27 +6,33 @@ import { compose } from 'redux';
 import get from 'lodash/get';
 import debounce from 'lodash/debounce';
 import isEmpty from 'lodash/isEmpty';
-import { Card, Skeleton, Input } from 'antd';
+import { Card, Skeleton, Input, Row, Col } from 'antd';
+
 import styled from 'styled-components';
 import { injectIntl } from 'react-intl';
 import { useHistory } from 'react-router-dom';
 import T from '@components/T';
 import Clickable from '@components/Clickable';
 import { useInjectSaga } from 'utils/injectSaga';
-import { selectHomeContainer, selectReposData, selectReposError, selectRepoName } from './selectors';
+import { selectHomeContainer, selectitunesData, selectitunesError, selectartistName } from './selectors';
```

Comment:
should be camelCase 

selectitunesData -> selectItunesData,
selectitunesError -> selectItunesError 
selectartistName -> selectArtistName